src/pages/nft/claim/index.vue

```diff
@@ -503,6 +529,7 @@ export default {
     if (this.isFreePurchase) {
       this.claimingAddress = this.claimingAddressInput;
     }
+    await this.fetchRecommendInfo();
```

Comment:
Should we refetch after use confirmed `claimingAddress` ?

Reply:
[🚸 Refetch recommendInfo after changing claimingAddress](https://github.com/likecoin/liker-land/pull/1532/commits/674ea554038d2941a1190a40144f7abbfc43ef7e)